tools/test-npm.sh

```diff
@@ -29,10 +29,10 @@ export npm_config_prefix="npm-prefix"
 export npm_config_tmp="npm-tmp"
 
 # install npm devDependencies and run npm's tests
-
-../$NODE cli.js install --ignore-scripts
-../$NODE cli.js run-script test-legacy
-../$NODE cli.js run-script test
+NODEPATH="$(../$NODE -e 'console.log(require("path").resolve(".."))')"
+PATH="$NODEPATH:$PATH" ../$NODE cli.js install --ignore-scripts
+PATH="$NODEPATH:$PATH" ../$NODE test/run.js
+PATH="$NODEPATH:$PATH" ../$NODE cli.js run-script tap -- "test/tap/*.js"
```

Comment:
I think this can just be declared as `export PATH="$NODEPATH:$PATH"` above, like https://github.com/nodejs/node/pull/3308/files#diff-de5d26632ba86cba492d3f4a7e6e86a9R29. This should only be locally exported to things the script calls to my understanding.